db/schema.yml

```diff
@@ -1232,6 +1232,7 @@ generic_objects:
 - generic_object_definition_id
 - created_at
 - updated_at
+- properties
```

Comment:
@bzwei For my own knowledge can you explain why this change is required?  We rarely edit this file when adding migrations so I am not sure why it is needed on this one.